src/third-party/text/2.65.0/manifest.json

```diff
@@ -5,6 +5,6 @@
     "dependencies": {
```

Comment:
Do you also need to update the version here (the line above)?

Reply:
Good catch